Formula/gh.rb

```diff
@@ -5,6 +5,8 @@ class Gh < Formula
   sha256 "49c42a3b951b67e29bc66e054fedb90ac2519f7e1bfc5c367e82cb173e4bb056"
   license "MIT"
 
```

Comment:
```suggestion
  revision 1
```
We need the revision so that existing users get a version with this update.

Reply:
Thank you, but I purposefully left the `revision` out from my PR, since it's not needed. The feature this enables is only present in the HEAD of the `cli/cli` repo and is slated for the next release. So I do *not* want the bottles for gh v1.5.0 to be rebuilt.

Reply:
So the new feature only works on `HEAD` builds? Seems like you should wrap the changes in an `if build.head?` block first, though I guess it's fine if they're a no-op for source builds from the current stable release.

If we're leaving out the `revision`, then let's be consistent with other formulae:
```suggestion
```

Reply:
@carlocab I have pushed the removal of the blank line.